metrics.md

```diff
@@ -0,0 +1,47 @@
+| gluster_brick_inodes_available | gluster_brick_inodes_available{volume="xxx", "brick"="/mnt/xxx", nodeid="xxxxx"} 100 |
+| gluster_brick_size_bytes | gluster_brick_size_bytes{volume="xxx", brick_path="xxxx", nodeid="xxx"} 1048576	|
+| gluster_brick_used_bytes|	gluster_brick_used_bytes{volume="xxx", brick_path="xxxx", nodeid="xxx"} 1048	|
+| gluster_brick_available_bytes |	gluster_brick_available_bytes{volume="xxx", brick_path="xxx", nodeid="xxx"} 1047576	|
+| gluster_disk_read_ios | gluster_disk_read_ios{brick_path="xxx",device="xxx",disk="vda",host="xxx",id="xxx",instance="xxx:8080",subvolume="xxx",volume="xxx"} 0
+|   gluster_disk_read_merges | gluster_disk_read_merges{brick_path="xxx",device="xxx",disk="vda",host="xxx",id="xxx",instance="xxx:8080",subvolume="xxx",volume="xxx"} 0
+|   gluster_disk_read_sectors | gluster_disk_read_sectors{brick_path="xxx",device="xxx",disk="vda",host="xxx",id="xxx",instance="xxx:8080",subvolume="xxx",volume="xxx"} 0
+|   gluster_disk_read_ticks | gluster_disk_read_ticks{brick_path="xxx",device="xxx",disk="vda",host="xxx",id="xxx",instance="xxx:8080",subvolume="xxx",volume="xxx"} 0
+|   gluster_disk_write_ios | gluster_disk_write_ios{brick_path="xxx",device="xxx",disk="vda",host="xxx",id="xxx",instance="xxx:8080",subvolume="xxx",volume="xxx"} 0
+|   gluster_disk_write_merges | gluster_disk_write_merges{brick_path="xxx",device="xxx",disk="vda",host="xxx",id="xxx",instance="xxx:8080",subvolume="xxx",volume="xxx"} 0 |
+|   gluster_disk_write_sectors | gluster_disk_write_sectors {brick_path="xxx",device="xxx",disk="vda",host="xxx",id="xxx",instance="xxx:8080",subvolume="xxx",volume="xxx"} 0 |
+|   gluster_disk_write_ticks | gluster_disk_write_ticks {brick_path="xxx",device="xxx",disk="vda",host="xxx",id="xxx",instance="xxx:8080",subvolume="xxx",volume="xxx"} 0 |
+|   gluster_disk_inflight | gluster_disk_inflight{brick_path="xxx",device="xxx",disk="vda",host="xxx",id="xxx",instance="xxx:8080",subvolume="xxx",volume="xxx"} 0 |
+|   gluster_disk_total_ticks | gluster_disk_total_ticks{brick_path="xxx",device="xxx",disk="vda",host="xxx",id="xxx",instance="xxx:8080",subvolume="xxx",volume="xxx"} 0 |
+|  gluster_disk_time_inqueue | gluster_disk_time_inqueue{brick_path="xxx",device="xxx",disk="vda",host="xxx",id="xxx",instance="xxx:8080",subvolume="xxx",volume="xxx"} 0 |
+
+#### Gluster process Metrics: 
+
+|Gluster Prometheus schema |example|
+|--------|-----------|
+| gluster_virtual_memory | gluster_virtual_memory{name="glusterfsd",nodeid="xxx",brick_path="/bricks/b1",volume="gv1"} 912260		
+| gluster_resident_memory | gluster_resident_memory{name="glusterfsd",nodeid="xxx",brick_path="/bricks/b1",volume="gv1"} 15392		
+| gluster_memory_percentage | gluster_memory_percentage{name="glusterfsd",nodeid="xxx",brick_path="/bricks/b1",volume="gv1"} 0.7		
+| gluster_cpu_percentage | gluster_cpu_percentage{name="glusterfsd",nodeid="xxx",brick_path="/bricks/b2",volume="gv1"} 0	
```

Comment:
One more which is supported now is `gluster_elapsed_time_seconds` (uptime)

Reply:
Metrics labels will change for each type of process. For example, brick details will not be available if `name=glusterd`

Reply:
ack.